hack/unwanted-dependencies.json

```diff
@@ -140,6 +140,9 @@
         "sigs.k8s.io/kustomize/api",
         "sigs.k8s.io/kustomize/kustomize/v5"
       ],
+      "github.com/xeipuuv/gojsonschema": [
+        "github.com/opencontainers/image-spec"
+      ],
```

Comment:
https://github.com/opencontainers/image-spec/pull/1189 has already removed the dependency and we can wait for the next release of `opencontainers/image-spec`.